### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'replace'

### ISSUE THREAD

**Issue by sbrl** [pirate/ArchiveBox#139]
**Describe the bug**
I've just updated to the latest version of ArchiveBox, and now it's throwing the exception below all the time.

**Steps to reproduce**
1. Checkout commit e6d5cd44327a93394e8ef892452ca526acac22cd
2. Execute `archivebox`
3. See error

**Screenshots or log output**
If applicable, use screenshots or copy/pasted terminal output to help explain your problem.

````
Traceback (most recent call last):
  File "./archive", line 184, in <module>
    links = merge_links(archive_path=out_dir, import_path=source, only_new=ONLY_NEW)
  File "./archive", line 78, in merge_links
    pretty_path(import_path),
  File "/path/to/ArchiveBox/archivebox/util.py", line 182, in pretty_path
    return path.replace(REPO_DIR + '/', '')
AttributeError: 'NoneType' object has no attribute 'replace'
````

**Software versions (please complete the following information):**
 - ArchiveBox version: e6d5cd44327a93394e8ef892452ca526acac22cd
 - Python version: 3.5.3
 - OS: Raspbian GNU/Linux 9.6 (stretch)
 - Chrome version: Not installed

@pirate

**Comment by pirate** [comment]
Fixed in latest `master` (74b99fe). Thanks!

**Comment by sbrl** [comment]
Awesome! I'll update now.
